lib/reply.js

```diff
@@ -216,6 +211,11 @@ function handleError (reply, error, cb) {
     return
   }
 
+  if (reply.res.headersSent) {
+    reply.res.log.info('response terminated with an error with headers already sent')
```

Comment:
reply.requesy.log is nicer imho

Reply:
I would change it, but `reply.res.log` is used just 20 lines above this one so it would look very strange to log in different ways in the same function.

Reply:
can you put `log` into its own variable then?

Reply:
Done 👍